templates/cosmic-settings.tera

```diff
@@ -0,0 +1,132 @@
+---
+name: "Catppuccin"
+window_hint_color: ""
+bg_alpha: 1.0
+frosted: false
+outer_gap_size: 0
+inner_gap_size: 8
+active_hint_size: 3
+whiskers:
+    version: 2.5.1
+    matrix:
+    - variant: ["round", "slightlyround", "square"]
+    - flavor
+    - accent
+    filename: "themes/{{ name }}{{ variant | capitalize }}-{{ flavor.identifier | capitalize }}-{{ accent | capitalize }}.ron"
```

Comment:
```suggestion
window_hint_color: ""
bg_alpha: 1.0
frosted: false
outer_gap_size: 0
inner_gap_size: 8
active_hint_size: 3
whiskers:
    version: 2.5.1
    matrix:
    - variant: ["round", "slightlyround", "square"]
    - flavor
    - accent
    filename: "themes/Catppuccin{{ variant | capitalize }}-{{ flavor.identifier | capitalize }}-{{ accent | capitalize }}.ron"
```

Reply:
I added the name field to make it easy for derivative themes to customize the name. Should it be patched in a derivative theme instead?

Reply:
> I added the name field to make it easy for derivative themes to customize the name. Should it be patched in a derivative theme instead?

I understand where you're coming from but I think this sets a bit of a weird precedent for the organisation where we allow our entire brand to be overridable. I'm all for customisation and it's obviously designed into whiskers with the way our colour overrides flag / frontmatter works but it feels odd to have that be overridable by default _within the Catppuccin organisation_.

Furthermore, I don't believe it makes any sense to only override "Catppuccin". If you were making an override for "MyAwesomeTheme" then your whiskers override would just result in "MyAwesomeTheme-Latte", "MyAwesomeTheme-Frappe", etc. It's also worth mentioning that including "Catppuccin" in the filename might not be required since there is a `name` field in the file itself which I'd imagine COSMIC uses as the display name on the UI.

With all of the above in mind, I'd push back on accepting the template with the frontmatter override and would either accept "Catppuccin" hardcoded into the filename or potentially not included in the first place. Hope that makes sense 👍